docs/patterns/tables.md

```diff
@@ -0,0 +1,66 @@
+# Tables
+
+| Benefits of using `<table>`                      | Provider                |
+|--------------------------------------------------|-------------------------|
+| Clean way of displaying tabular data             | Browser                 |
+| Great browser support                            | Browser                 |
+| Can copy paste the table into other applications | Browser                 |
+| Can reorder items in the table!                  | `react-beautiful-dnd` 😎|
+
+`react-beautiful-dnd` supports requires no additional wrapping elements to create `Draggable` and `Droppable` components. Therefore it is possible to have a `<table>` that has valid HTML as well as supporting drag and drop.
+
+> We have not found a way to achieve semantic reordering of table columns at this stage. This is because there is no one element that represents a table column - rather, a column is a result of cell placements within repeating rows. As such as cannot wrap a `Draggable` around a 'column' in order to make it draggable. PR's to this guide are welcome if you find a working approach!
+
+## Strategies
+
+There are two strategies you can use when reordering tables.
+
+1. Fixed layouts (faster and simplier)
+2. Dimension locking (slower but more robust)
+
+### Strategy 1: fixed layouts
+
+In order to use this strategy the widths of your columns need to be fixed - that is, they will not change depending on what content is placed in the cells. This can be achieve with either a `table-layout: fixed` or `table-layout: auto` as long as you manually set the width of the cells (eg `50%`).
+
+The only thing you need to do is set `display: table` on a `Draggable` row while it is dragging.
+
+[See example code here](TODO)
```

Comment:
Do we have code samples for here, [here](https://github.com/atlassian/react-beautiful-dnd/pull/406/files#diff-8029e14a5c1d3dab700f1fc881dcedafR42) and [here](https://github.com/atlassian/react-beautiful-dnd/pull/406/files#diff-8029e14a5c1d3dab700f1fc881dcedafR66)?

Reply:
We do now! 👍

Reply:
I'll update the links